index.d.ts

```diff
@@ -8,15 +8,10 @@
 //   Michal Adamczyk <https://github.com/mradamczyk>
 //   Marvin Hagemeister <https://github.com/marvinhagemeister>
 
-export type ClassValue = string | number | ClassDictionary | ClassArray | undefined | null | boolean;
+export type Value = string | number | boolean | undefined | null;
```

Comment:
I don't know if `Value` is a nice name to be honest.

These types aren't exactly capturing the subtlety of `string` arguments, compared to value truthiness.